planemo/commands/cmd_workflow_track.py

```diff
@@ -0,0 +1,26 @@
+"""Module describing the planemo ``workflow_track`` command."""
+
+import click
+
+from planemo import options
+from planemo.cli import command_function
+from planemo.engine.factory import engine_context
+from planemo.galaxy.activity import wait_for_invocation_and_jobs
+
+
+@click.command("workflow_track")
+@options.invocation_target_options()
+@command_function
+def cli(ctx, invocation_id, **kwds):
+    """Run defined tests against existing workflow invocation."""
```

Comment:
I assume this is a copy and paste artifact?

Reply:
I think so, fixed this now.